modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -17,7 +17,7 @@ You can install the AWS Load Balancer Operator from the OperatorHub by using the
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
 . Select the `aws-load-balancer-operator` namespace.
-. Follow the instructions to prepare the Operator for installation.
+. Follow the instructions to prepare the cluster VPC and subnets for the Operator installation.
```

Comment:
Since we are using VPC (virtual private cloud??) for the first time in this file, we need to specify the long form. For example: 
`Follow the instructions to prepare the cluster virtual private cloud (VPC) and subnets for the Operator installation.`